src/department/department.controller.ts

```diff
@@ -38,6 +45,18 @@ export class DepartmentController {
   ): Promise<DepartmentEntity> {
     return this.departmentServise.createDepartment(createDepartmentDto);
   }
+  @Post(":id/employees")
```

Comment:
how will the route look like if you need to add multiple employees then?

Reply:
Employees can only be added one at a time

Reply:
as for "today"